### PR TITLE
add error info for dnet

### PR DESCRIPTION
**Pull request by xrdavies**
* add error info for dnet
* add help info for dnet
* reformat dnet_xdag.c

### CODE REVIEW

File: dnet/dnet_xdag.c [hunk 1]
```diff
@@ -698,7 +847,11 @@ int dnet_execute_command(const char *cmd, void *fileout)
 		struct sockaddr_in peeraddr;
 		int fd;
 		str = strtok_r(0, " \t\r\n", &lasts);
-		if(!str) { fprintf(f, "connect: parameter is absent\n"); return -1; }
+		if(!str) {
+			fprintf(f, "connect: parameter is absent\n");
```
Comment (sgaragagghu) [inline dnet/dnet_xdag.c]: maybe you can print help in this case too

File: dnet/dnet_xdag.c [hunk 2]
```diff
@@ -355,46 +430,69 @@ static void *xthread_main(void *arg)
...
-							if(!conn->crypt) { err = 6; goto close; }
+							if(!conn->crypt) {
+								err = 6;
+								goto close;
+							}
 							conn->crypt->last_sent = time(0);
 						}
-					}
-					if(conn->crypt) {
+					} else {
```
Comment (sgaragagghu) [inline dnet/dnet_xdag.c]: conn-crypt is modified
`							conn->crypt = (struct xcrypt *)malloc(sizeof(struct xcrypt));
`
that's why there is 					`if(conn->crypt) {
`
and not` } else {`

Comment (jonano614) [reply dnet/dnet_xdag.c]: Ah, @sgaragagghu I did't see your comments. Yes, it is a good point

Comment (xrdavies) [reply dnet/dnet_xdag.c]: Thank you @sgaragagghu

**Pull request merged**